### PR TITLE
Added minimal cmake configuration

### PR DESCRIPTION
**Pull request by tjanc**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,42 @@
+cmake_minimum_required(VERSION 3.4 FATAL_ERROR)
+
+project(sundown-apiary VERSION 1.0 LANGUAGES C)
+
+add_library(sundown-apiary
+        src/autolink.c
```
Comment (kylef) [inline CMakeLists.txt]: I noticed here you are using 8-spaces indentation, but in install and target_include_directories 4 spaces. Is that a mistake or is this intentional/normal in cmake?

File: CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,42 @@
+cmake_minimum_required(VERSION 3.4 FATAL_ERROR)
+
+project(sundown-apiary VERSION 1.0 LANGUAGES C)
+
+add_library(sundown-apiary
```
Comment (kylef) [inline CMakeLists.txt]: Looking at drafter.gyp libsundown target. There is also building the files in `html` and exposing them as headers. I take it these are entirely unused by Snowcrash/Drafter?

File: CMakeLists.txt (hunk 1 above)
Comment (tjanc) [reply CMakeLists.txt]: Yeah, that's a mistake.

File: CMakeLists.txt (hunk 2 above)
Comment (tjanc) [reply CMakeLists.txt]: Looks like it. Drafter/snowcrash builds, links and executes all tests successfully.

**Pull request merged**